src/amp/server/document.test.tsx

```diff
@@ -56,12 +59,17 @@ test('produces valid AMP doc', async () => {
         },
     };
 
+    const activeTests = extractModelAndStyle(
+        getActiveTests(abTestsFullConfig, {"ab-zero-test-experiment": true})
+    )[0];
+
     const body = (
         <Article
             nav={nav}
             articleData={CAPI}
             config={config}
             analytics={analytics}
+            experiment={activeTests}
```

Comment:
assigning something plural to something singular confuses me :)

Reply:
Changed naming conventions in refactor